openproblems/tasks/cell_cell_communication/methods/natmi.py

```diff
@@ -0,0 +1,20 @@
+from ....tools.decorators import method
+from .liana import liana
```

Comment:
This method can probably just be moved to `liana.py`, and then they can also share a decorator partial.